contracts/multiswap-contracts/FundManager.sol

```diff
@@ -20,8 +20,13 @@ contract FundManager is SigCheckable, LiquidityManagerRole {
         );
     bytes32 constant WITHDRAW_SIGNED_WITH_SWAP_METHOD =
```

Comment:
same

Reply:
It was updated in recent commit few weeks ago.